target/linux/qualcommax/image/ipq60xx.mk

```diff
@@ -24,19 +24,25 @@ define Device/cambiumnetworks_xe3-4
 endef
 TARGET_DEVICES += cambiumnetworks_xe3-4
 
-define Device/linksys_mr7350
+define Device/linksys_mr
 	$(call Device/FitImage)
 	DEVICE_VENDOR := Linksys
-	DEVICE_MODEL := MR7350
-	SOC := ipq6000
-	NAND_SIZE := 256m
-	KERNEL_SIZE := 8192k
 	BLOCKSIZE := 128k
 	PAGESIZE := 2048
-	IMAGE_SIZE := 75776k
+	KERNEL_SIZE := 8192k
+	NAND_SIZE := 512m
+	IMAGE_SIZE := 147456k
```

Comment:
Please dont set these if you are going to have to override them later, they are not common then